.gitignore

```diff
@@ -13,5 +13,6 @@ gemfile.lock
 
 .rvmrc
 .ruby-version
+Gemfile.lock
```

Comment:
nit: I can't test this now but I see a lowercase `gemfile.lock` a few lines up.
Is that a bug from the bundler generator? Are we sure we need both?

Reply:
If we are ignoring the lockfile, then `gems.rb` is another name which Bundler supports, and can be told to create.

Reply:
> nit: I can't test this now but I see a lowercase `gemfile.lock` a few lines up.
> Is that a bug from the bundler generator? Are we sure we need both?

Didn't notice that, thank you.

BTW, gitignore is case sensitive by default, although it can be configured: https://stackoverflow.com/q/11116636/2630849

But I'd stick with default for people. 



> If we are ignoring the lockfile, then `gems.rb` is another name which Bundler supports, and can be told to create.

I didn't face with such file (it hasn't became default for Bundler 2), but isn't it an alternative for `Gemfile`, not a lock-file? I don't know a lock-file name for `gems.rb`.

Reply:
BTW I changed the commit and just "fixed" casing for `Gemfile.lock`, thank you.